src/uct/ib/mlx5/dc/dc_mlx5.h

```diff
@@ -331,6 +331,9 @@ struct uct_dc_mlx5_iface {
         uint8_t                      av_fl_mlid;
 
         uint8_t                      num_dci_channels;
+
+        /* used in hybrid dcs policy otherwise -1 */
+        int8_t                       hybrid_hw_dci;
```

Comment:
are we limited to 127 DCIs? maybe make it uint8_t?

Reply:
Maybe it should be uint16_t because I don't want this value to intersect with `UCT_DC_MLX5_EP_NO_DCI` which is a `(uint8_t) -1`

Reply:
ok